src/nested_pandas/utils/utils.py

```diff
@@ -25,6 +23,32 @@ def count_nested(df, nested, by=None, join=True) -> NestedFrame:
     Returns
     -------
     NestedFrame
+
+    Examples
+    --------
+
+    >>> from nested_pandas.datasets.generation import generate_data
+    >>> nf = generate_data(5,10,seed=1)
+
+    >>> from nested_pandas.utils import count_nested
+    >>> count_nested(nf, "nested")
+              a         b                                             nested  n_nested
+    0  0.417022  0.184677  [{t: 8.38389, flux: 10.233443, band: 'g'}; …] ...        10
+    1  0.720324  0.372520  [{t: 13.70439, flux: 41.405599, band: 'g'}; …]...        10
+    2  0.000114  0.691121  [{t: 4.089045, flux: 69.440016, band: 'g'}; …]...        10
+    3  0.302333  0.793535  [{t: 17.562349, flux: 41.417927, band: 'g'}; …...        10
+    4  0.146756  1.077633  [{t: 0.547752, flux: 4.995346, band: 'r'}; …] ...        10
+
+    `count_nested` also allows counting by a given subcolumn, for example we
+    can count by "band" label:
+
+    >>> count_nested(nf, "nested", by="band")
```

Comment:
This is too wide for the rendered page. Best suggestion I have is to use `display.float_format` and truncate all those floating points 1-2 decimals to make the display not so wide.

![image](https://github.com/user-attachments/assets/4010643a-0c23-45ce-9fee-e85f0416962b)

It's nice that the display gives you a slider, but better not to need it.

Reply:
Good call, I changed this to instead use "join=False" to just return a small nestedframe result instead to avoid it and teach about the join kwarg.